main.go

```diff
@@ -335,9 +343,20 @@ func mainerr() error {
 			case *fVersion:
 				fmt.Printf("%v %v\n", mp.Module.Path, mp.Module.Version)
 			case *fRun:
-				argv := append([]string{target}, runArgs...)
-				if err := syscall.Exec(argv[0], argv, os.Environ()); err != nil {
-					return fmt.Errorf("failed to exec %v: %v", target, err)
+				if runtime.GOOS == "windows" {
+					run := exec.Command(target, runArgs...)
+					run.Stdout = os.Stdout
+					run.Stderr = os.Stderr
+					if err := run.Run(); err != nil {
+						if _, ok := err.(*exec.ExitError); !ok {
+							return fmt.Errorf("failed to run %v: %v", strings.Join(run.Args, " "), err)
+						}
+					}
+				} else {
+					argv := append([]string{target}, runArgs...)
+					if err := syscall.Exec(argv[0], argv, os.Environ()); err != nil {
```

Comment:
I wonder - if `gobin` is supposed to be portable, why not just use `exec.Command` on all platforms? I know that an exec is better when available, but I'd imagine that we care more about consistency.

Reply:
I pondered this too. But I reasoned that ideally we push for `exec` and fall back to an alternative if not available. Reasoning was along the lines of what you see in `ps` making more sense. But I can see an argument both ways. Let's give this a try for now?